src/tests/0000ztrips.js

```diff
@@ -138,4 +158,53 @@ describe('trips tests', () => {
         done();
       });
   });
+  it('manager should get request made by his/her own direct', (done) => {
+    chai.request(app).get('/api/v1/trip/trip-requests')
+      .set('token', `Bearer ${token2}`)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.an('object');
+        chai.expect(res.body.message).to.eq('Trips requested by your direct reports');
+        done();
+      });
+  });
+  it('manager should get requests on the selected page', (done) => {
+    chai.request(app).get('/api/v1/trip/trip-requests?page=1')
+      .set('token', `Bearer ${token2}`)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.an('object');
+        chai.expect(res.body.message).to.eq('Trips requested by your direct reports');
+        done();
+      });
+  });
+  it('should get error when a page is requesting dont have data', (done) => {
+    chai.request(app).get('/api/v1/trip/trip-requests?page=2')
+      .set('token', `Bearer ${token2}`)
+      .end((err, res) => {
+        res.should.have.status(404);
+        res.body.should.be.an('object');
+        chai.expect(res.body.error).to.eq('No trip requests on this page');
+
+        done();
+      });
+  });
+  it('paginating tests', (done) => {
```

Comment:
@mujohn26 Kindly provide a concise and descriptive test case description. It is not intuitive to know with certainty what you're testing here.

Reply:
I resolved it.